.github/workflows/deploy.yml

```diff
@@ -0,0 +1,80 @@
+name: Build Docker and Deploy on Kubernetes
+on:
+  push:
+    branches:
+      - main
+      - feature/dev_deploy
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    needs: tests
+    env:
+      SERVICES: relay,relay-ws
+      APP_NAME: rpc-relay
+      ENVIRONMENT: dev
+      REGISTRY: ${{ secrets.ECR_REGISTRY }}
+      K8S_VERSION: v1.25.0
+      DOCKER_BUILDKIT: 1
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Declare variables
+        id: vars
+        shell: bash
+        run: |
+          echo "version=$(jq -r '.info.version' docs/openrpc.json)" >> "$GITHUB_OUTPUT"
+          echo "image_tag=${{ env.ENVIRONMENT }}-$(git rev-parse --short HEAD)" >> "$GITHUB_OUTPUT"
+
+      - name: Set Docker image
+        run: |
+          for service in ${{ env.SERVICES }}; do
```

Comment:
Perhaps it would be easier to use env var(s) inside the `docker-compose.yml` instead of writing changes 🤔